MainModule/Server/Commands/Moderators.lua

```diff
@@ -6627,13 +6627,19 @@ return function(Vargs, env)
 			Description = "AI bots made for training; ':bot scel 5 true true'";
 			AdminLevel = "Moderators";
 			Function = function(plr: Player, args: {string})
-				local num = tonumber(args[2]) and math.max(tonumber(args[2]), 50) or 1
+				local num = tonumber(args[2]) and math.min(tonumber(args[2]), 50) or 1
 				local health = tonumber(args[6]) or 100
 				local speed = tonumber(args[7]) or 16
 				local damage = tonumber(args[8]) or 5
-				local walk = args[3] == "false" and false or true
```

Comment:
revert this change. why'd you make this worse?

Reply:
you'll probably have to expand the code view to see what you've done. there is programattically no reason as to why this should have been done

Reply:
Didn't knew `wait` was replaced with `task.wait` by default, I may revert those


If you are talking about line 6634, it always evaluates to true as `x and false or y` will always be y, even Roblox Studio gives an error about it. You can test it if you want, the bots will walk even if you give false

If you are talking about line 6630, I don't see any reason why math.max would be used there. You can also test if this is right, as the bot number would never be lower than 50 if math.max was used


I tested the loader, :bot (with walk as true and false), :alert, :cd, :pchat and :vote, and !ping (I don't know if that matters)

Reply:
alright

Reply:
@Expertcoderz head hurty